frontend/src/compliance_reporting/ScheduleSummaryContainer.js

```diff
@@ -397,14 +388,19 @@ class ScheduleSummaryContainer extends Component {
       };
 
       penalty = ScheduleSummaryContainer.calculateNonCompliancePayable(penalty);
-
-      this.setState({
-        diesel,
-        gasoline,
-        part3,
-        penalty
-      });
     }
+
+    console.error(diesel);
+    console.error(gasoline);
```

Comment:
intentional?

Reply:
thanks for catching this. should be gone now